tests/Cosmos/StakingTests.cpp

```diff
@@ -162,16 +164,17 @@ TEST(CosmosStaking, Withdraw) {
     ASSERT_EQ(hex(output.signature()), "546f0d67356f6af94cfb5ab22b974e499c33123f2c2c292f4f0e64878e0e728f4643105fd771550beb3f2371f08880aaa38fa8f2334c103a779f1d82d2db98d6");
 }
 
-TEST(CosmosStaking, WithdrawAll) {
+TEST(CosmosStaking, WithdrawAllRaw) {
```

Comment:
should we remove `MsgWithdrawDelegationRewardsAll `as it's not implemented on cosmos sdk side?

Reply:
It is removed, this test shows how to sign arbitrary msg

Reply:
see https://github.com/trustwallet/wallet-core/pull/766/files#diff-a6612f424546729f3327003344e6698dR174-R177